components/raftstore/src/store/peer.rs

```diff
@@ -1211,7 +1211,7 @@ where
             > 0
         {
             self.raft_group.raft.set_max_apply_unpersisted_log_limit(0);
-            RAFT_DISABLE_UNPERSISTED_APPLY_GAUGE.inc();
+            RAFT_ENABLE_UNPERSISTED_APPLY_GAUGE.dec();
```

Comment:
If one peer is removed, `dec()` wouldn't be called.

Reply:
fixed, add try dec in `on_destroy_peer`.